init/Kconfig

```diff
@@ -1927,14 +1927,27 @@ config HAS_CARGO
 config HAS_CARGO_XBUILD
   def_bool $(success,cargo xbuild --version)
 
-config RUST
+config MENU_RUST
   bool "Enables building kernel modules written in Rust"
   depends on HAS_RUST
```

Comment:
Seems like we only need to know if "cargo xbuild" works, and that sufficient for "HAS_RUST" in the sense that there's nothing we can do with only the existing HAS_RUST nor HAS_CARGO. Only HAS_CARGO_XBUILD is meaningful (it requires rustc and cargo), and the kernel needs full 'cargo xbuild' support.

Reply:
`cargo xbuild` isn't even required anymore -- as of a week or two ago, everything we need is in upstream cargo!

Reply:
I wonder whether we need to check for any of this. If Rust modules get into mainline, we will build them as any other, so we will assume we have `rustc` etc. available no matter what (and we will want to be able to change it like we do with `CC`). And if there are no modules, we don't care anyway.